### PR TITLE
Scale widget update

### PR DESCRIPTION
**Pull request by povilasRi**
Atnaujintas esrio widget css

### CODE REVIEW

**Comment by anged** [comment]
What is exactly updated?

**Comment by povilasRi** [comment]
Removed old style for bottom left corner and updated according to new esri widget styles.

File: src/styles/_main.scss [hunk 1]
```diff
@@ -1369,12 +1369,10 @@ measure-map {
 #vilnius-logo img {
   width: 40px;
 }
-
-.esri-scale-bar__label {
-  text-shadow: 0 0 1px #343131, 0 0 1px #343131, 0 0 1px #343131;
-  color: #fff;
+.esri-ui-bottom-left {
```
Comment (anged) [inline src/styles/_main.scss]: Ok, remove old styling from main.scss for desktop view

Comment (anged) [reply src/styles/_main.scss]: Plus adapt styling for esri widget ui, based on old maps.vilnius.lt apllication

**Comment by anged** [comment]
Pull request shoul be added to development repo

**Pull request closed without merging**